flake.nix

```diff
@@ -23,6 +23,11 @@
     # actually not used when using the modules but than nothing ever will try to fetch this nixpkgs variant
     srvos.inputs.nixpkgs.follows = "nixpkgs";
 
+    buildbot-nix.url = "github:Mic92/buildbot-nix";
```

Comment:
I would maybe pin this in the beginning to a commit so that flake auto-updates don't break stuff. And then we live-test upgrades. I am still working on rolling this out in other places.